### DISCUSSION TITLE
Download cleanlog images?

### DISCUSSION THREAD

**Discussion by ghost** [mrbungle64/ecovacs-deebot.js#273] [Q&A] [answered]
Hi,

Is there a way to download the CleanLog images? Downloading them from the App2.js (using https) results in a file only containing the text '{"code":3001,"message":"sign check unpassed or userid was empty"}' while I hoped sinse the app a signed in the download would work.

**Comment by mrbungle64** [comment, marked as answer]
@cactusss 

You can use the following async function:

```js
vacbot.downloadSecuredContent(url, targetFilename);
```

**Comment by ghost** [comment]
Excelent! Thanks Sascha.
